packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "24.0.1",
+  "version": "25.0.0",
```

Comment:
I am incrementing core because I will be creating a new release directly after this is merged. This is a major release because we had a React component deprecation merged before this.